frontend/src/app/form-control-validation.directive.ts

```diff
@@ -3,11 +3,13 @@ import { Directive, HostBinding, Optional } from '@angular/core';
 import { NgControl } from '@angular/forms';
 
 @Directive({
-  selector: '.form-control:not([ngbNoFormControlValidation]),[ngbFormControlValidation]'
+  selector: '.form-control:not([glFormControlValidation]),[glFormControlValidation],[ngbRadioGroup]'
 })
 export class FormControlValidationDirective {
 
-  constructor(@Optional() private ngControl: NgControl) { }
+  constructor(@Optional() private ngControl: NgControl) {
+    console.log(`ngControl = ${ngControl}`);
```

Comment:
remains a console.log

Reply:
Thanks. FYI, I files a PR for ng-bootstrap, because ngbRadioGroup never becomes touched. I just realized it's alreaby been merged, so it will be part of the next release: https://github.com/ng-bootstrap/ng-bootstrap/pull/1988